src/client/dfs/duns.c

```diff
@@ -1086,6 +1131,194 @@ duns_create_path(daos_handle_t poh, const char *path, struct duns_attr_t *attrp)
 	return rc;
 }
 
+int
+duns_link_cont(daos_handle_t poh, const char *cont, const char *path)
+{
+	daos_handle_t		coh;
+	daos_prop_t		*prop;
+	struct daos_prop_entry	*entry;
+	daos_pool_info_t	pinfo = {0};
+	daos_cont_info_t	cinfo = {0};
+	daos_cont_layout_t	type;
+	char			pool_str[37], cont_str[37];
```

Comment:
Is this what `DAOS_UUID_STR_SIZE` is for?
```suggestion
	char			pool_str[DAOS_UUID_STR_SIZE], cont_str[DAOS_UUID_STR_SIZE];
```

Reply:
sure this looks better. ill update if re-push is needed. although not too concerning as uuids will never change length :-)